src/Illuminate/Collections/Arr.php

```diff
@@ -817,4 +817,21 @@ public static function wrap($value)
 
         return is_array($value) ? $value : [$value];
     }
+
+    /**
+     * Insert new item in array on any position.
+     *
+     * @param  array  $array
+     * @param  int  $pos
+     * @param $value
+     * @return array
+     */
+    public static function insert(array &$array, int $pos, $value)
+    {
+        return array_merge(
+            array_slice($array, 0, $pos),
+            static::wrap($value),
+            array_slice($array, $pos, null, true)
+        );
```

Comment:
Wouldn't `array_splice()` work here?

~~~php
public static function insert(array &$array, int $pos, $value)
{
    array_splice($array, $pos, 0, static::wrap($value));

    return $array;
}
~~~

https://www.php.net/manual/en/function.array-splice.php

Reply:
apply it